### ISSUE TITLE
Unexpected character encountered while parsing value

### ISSUE THREAD

**Issue by phirestalker** [rogerfar/rdt-client#58]
I got this error on one of my torrents.
```
[15:30:00 ERR] Unexpected error occurred in TorrentDownloadManager.Tick
Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: <. Path '', line 0, position 0.
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonReader.ReadForType(JsonContract contract, Boolean hasConverter)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
   at RDNET.RdNetClient.Get[T](String url, Boolean repeatRequest)
   at RDNET.RdNetClient.GetTorrentInfoAsync(String id)
   at RdtClient.Service.Services.Torrents.Update() in /appserver/server/RdtClient.Service/Services/Torrents.cs:line 374
   at RdtClient.Service.Services.TorrentRunner.Tick() in /appserver/server/RdtClient.Service/Services/TorrentRunner.cs:line 202
   at RdtClient.Service.Services.TaskRunner.ExecuteAsync(CancellationToken stoppingToken) in /appserver/server/RdtClient.Service/Services/TaskRunner.cs:line 36
```

**Comment by rogerfar** [comment]
That looks like a malformed response from real-debrid's API.

What does the torrent in real debrid say? And what happens if you re-add it?

**Comment by phirestalker** [comment]
The torrents were all automatically added so I am not sure which one it is. This only happened once, so I imagine it was a fluke from real-debrid.

**Comment by minus1kelvin** [comment]
I'm having this issue fairly consistently now. Restarting the docker container seems to fix but before that it will consistently fail to download anything.

**Comment by rogerfar** [comment]
Can you try enabling the debug log from the settings panel, then send last part of the logs when this happens?

**Comment by minus1kelvin** [comment]
Sure! I'll update if/when it happens again. Thanks!

**Comment by rogerfar** [comment]
I added some extra error handling for deserialization errors from the API, when it happens again it should report the actual issue. This fix is added to 1.7.5.
